test/Microsoft.AspNet.OData.Versioning.ApiExplorer.Tests/Microsoft.AspNet.OData.Versioning.ApiExplorer.Tests.csproj

```diff
@@ -1,23 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This file doesn't seem to have actually been changed. It appears that only the extra lines have been removed. It's not a huge deal, but there's not much sense in changing a file that doesn't need changing.